internal/exec/help.go

```diff
@@ -35,6 +35,24 @@ func processHelp(
 		u.PrintMessage(fmt.Sprintf("atmos %s <subcommand> <component> -s <stack> [options]", componentType))
 		u.PrintMessage(fmt.Sprintf("atmos %s <subcommand> <component> --stack <stack> [options]", componentType))
 		u.PrintMessage(fmt.Sprintf("\nFor more details, execute '%s --help'\n", componentType))
+	} else if componentType == "terraform" && command == "clean" {
+		u.PrintMessage("\n'atmos terraform clean' command deletes the following folders and files from the component's directory:\n\n" +
+			" - '.terraform' folder\n" +
+			" - folder that the 'TF_DATA_DIR' ENV var points to\n" +
+			" - '.terraform.lock.hcl' file\n" +
+			" - generated varfile for the component in the stack\n" +
+			" - generated planfile for the component in the stack\n" +
+			" - generated 'backend.tf.json' file\n" +
+			" - 'terraform.tfstate.d' folder (if '--everything' flag is used)\n\n" +
+			"Usage: atmos terraform clean <component> -s <stack> <flags>\n\n" +
+			"Use '--everything' flag to also delete the Terraform state files and and directories with confirm message.\n\n" +
+			"Use --force to forcefully delete Terraform state files and directories for the component.\n\n" +
+			"- If no component is specified, the command will apply to all components and stacks.\n" +
+			"- If no stack is specified, the command will apply to all stacks for the specified component.\n" +
+			"Use '--skip-lock-file' flag to skip deleting the '.terraform.lock.hcl' file.\n\n" +
+			"If no component or stack is specified, the clean operation will apply globally to all components.\n\n" +
+			"For more details refer to https://atmos.tools/cli/commands/terraform/clean\n")
+
```

Comment:
I think we're regressing here. This was heavily refactored in https://github.com/cloudposse/atmos/pull/914

Reply:
We should go away from this file. Is there something we are missing that the code handle comes here?